modules/apps/portal-background-task/portal-background-task-service/src/main/java/com/liferay/portal/background/task/internal/settings/definition/BackgroundTaskManagerConfigurationBeanDeclaration.java

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.portal.background.task.internal.settings.definition;
+
+import com.liferay.portal.background.task.configuration.BackgroundTaskManagerConfiguration;
+import com.liferay.portal.kernel.settings.definition.ConfigurationBeanDeclaration;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Vendel Toreki
+ */
+@Component(service = ConfigurationBeanDeclaration.class)
+public class BackgroundTaskManagerConfigurationBeanDeclaration
+	implements ConfigurationBeanDeclaration {
+
+	@Override
+	public Class<?> getConfigurationBeanClass() {
+		return BackgroundTaskManagerConfiguration.class;
+	}
+
+}
```

Comment:
@vendeltoreki, this and LayoutSetPrototypeSystemConfigurationBeanDeclaration are unnecessary. Since you didn't use ConfigurationProvider to get Configuration.